init.c

```diff
@@ -2062,6 +2062,16 @@ static int __parse_jobs_ini(struct thread_data *td,
 		}
 
 		ret = fio_options_parse(td, opts, num_opts);
+
+		if (!ret) {
+			if (!strcmp(file, "-") && td->o.read_iolog_file != NULL) {
+				char *fname = get_name_by_idx(td->o.read_iolog_file, td->subjob_number);
```

Comment:
Maybe wrap this at the command and align the line below to the bracket.

Reply:
Done.